src/app/cube/home/splash/splash.component.scss

```diff
@@ -7,10 +7,7 @@
     background: linear-gradient(0deg, $primary-white 0%, rgba(255, 255, 255, 0) 100%);
     border: 1px solid $secondary-white;
     border-top: none;
-
-    //remove after 5th birthday
-    margin-top: 100px;
-    //remove after 5th birthday
+    padding-top: 135px;
```

Comment:
I'd add a "remove after 5th birthday comment" so this doesn't get missed when we remove the banner.